app/webpacker/controllers/checkbox_display_controller.js

```diff
@@ -0,0 +1,37 @@
+import { Controller } from "stimulus";
+
+export default class extends Controller {
+  static values = { target: String };
+
+  connect() {
+    this.element.addEventListener("change", this.change.bind(this));
+    if (this.element.checked) {
+      this.showTarget();
+    } else {
+      this.hideTarget();
+    }
+  }
+
+  disconnect() {
+    this.element.removeEventListener("change", this.change.bind(this));
+  }
+
+  change(event) {
+    // if the checkbox is checked, display the target
+    if (this.element.checked) {
+      this.showTarget();
+    } else {
+      this.hideTarget();
+    }
+  }
+
+  showTarget() {
+    let target = document.getElementById(this.targetValue);
```

Comment:
`±` 
`targetValue` is an ID, so I would rename to `targetId`

Reply:
I've changed to `target-id` (in html), `targetId` (in stimulus controller) and then `targetIdValue` (to get its value). 

--> https://stimulus.hotwired.dev/reference/values